facia/conf/routes

```diff
@@ -18,6 +18,8 @@ GET        /.well-known/security.txt.asc
 
 GET        /.well-known/amphtml/apikey.pub                                          controllers.FaciaController.ampRsaPublicKey()
 
+GET        /.well-known/atproto-did.txt                                             controllers.Assets.at(path="/public", file="atproto-did.txt")
```

Comment:
The docs don't specify a .txt extension. Do you think this will be okay?

https://blueskyweb.zendesk.com/hc/en-us/articles/19001802873101-How-to-Set-your-Domain-as-your-Handle

Reply:
That's a good point. I'll update.